src/BEditaClient.php

```diff
@@ -305,6 +305,23 @@ public function removeRelated($id, string $type, string $relation, array $data,
         return $this->delete(sprintf('/%s/%s/relationships/%s', $type, $id, $relation), json_encode($body), $headers);
     }
 
+    /**
+     * Replace a list of related objects: previuosly related are removed and replaced with these.
+     *
+     * @param int|string $id Object id
+     * @param string $type Object type name
+     * @param string $relation Relation name
+     * @param string $data Related objects to remove from relation
```

Comment:
I guess it's the opposite: `Related objects to apply by relation (replace previously related data)`

Reply:
Right. They can also be `resources`
Comment updated, also previous ones